FHEM/00_SIGNALduino.pm

```diff
@@ -4752,9 +4752,9 @@ USB-connected devices (SIGNALduino):<br>
     Default is to not to use long IDs for all devices.
     <br><br>
     Examples:<PRE>
-      # Do not use any long IDs for any devices:
+      # Do not use any long IDs for any devices (this is default, except OREGON):
```

Comment:
Was hältst Du davon wenn in der commandref die drei möglichen Werte
Unset, 0,1,Regexp definiert werden.

Bei Unset wird das Verhalten durch das Modul festgelegt. Am Beispiel Oregon werden longids verwendet.

Bei 0 werden keine longids verwendet 
usw.

Reply:
Das steht doch eigentlich schon drin:

```
      # Keine langen IDs verwenden (Voreinstellung):
      attr sduino longids 0
      # Immer lange IDs verwenden:
      attr sduino longids 1
      # Verwende lange IDs für SD_WS07 Devices.
      # Device Namen sehen z.B. so aus: SD_WS07_TH_3.
      attr sduino longids SD_WS07_TH
```

Reply:
Den Unterschied zwisch nicht definiert und 0 sehe ich nicht

```
# Keine langen IDs verwenden (Voreinstellung, außer OREGON):
      attr sduino longids 0
```

Das Attribut als 0 ist leider nicht die Voreinstellung, sondern bei 0 verwendet auch das Oregon Modul keine longids

Reply:
Naja, ich dachte Voreinstellung=default=undefiniert ...
Schreib einfach mal, was du gerne sehen möchtest.

Reply:
> Naja, ich dachte Voreinstellung=default=undefiniert ... Schreib einfach mal, was du gerne sehen möchtest.

Ich lese das anders ;(


      # Voreinstellung, Nutzung von langen IDs wird durch logische Module definiert. Meist deaktiviert, außer bei OREGON:
      delattr sduino longids 
      # keine lange IDs verwenden:
      attr sduino longids 0
      # Immer lange IDs verwenden:
      attr sduino longids 1
      # Verwende lange IDs für SD_WS07 Devices.
      # Device Namen sehen z.B. so aus: SD_WS07_TH_3.
      attr sduino longids SD_WS07_TH